.github/workflows/test.yml

```diff
@@ -18,6 +18,7 @@ jobs:
       - uses: actions/setup-node@v2
         with:
           node-version: '16'
+          cache: 'npm'
```

Comment:
yeah, do you have numbers? That's caching the entire install which should save us a few minutes, right ?

Reply:
total time of the workflow right now is 7-8 minutes, install time is 1-2 minutes. so we probably save around 15% of time